src/dagre/position/bk.js

```diff
@@ -71,7 +71,7 @@ function findType1Conflicts(g, layering) {
     return layer;
   }
 
-  layering.reduce(visitLayer);
+  if (Array.isArray(layering) && layering.length > 0) layering.reduce(visitLayer);
```

Comment:
I think it's fine to skip `Array.isArray()`.

I had a look at the code and `layering` comes from:

https://github.com/tbo47/dagre-es/blob/e03371f2ee986cd30ac5b3229e18816980e8fb7d/src/dagre/util.js#L135-L145

which always returns an array (although, sometimes an empty array).